Ginger/Ginger/Actions/ActionEditPage.xaml.cs

```diff
@@ -454,7 +453,7 @@ private void InitOutputValuesTabView()
             if (!datasourceGridToolbarItemsAdded)
             {
                 datasourceGridToolbarItemsAdded = true;
-                xDataSourceConfigGrid.AddToolbarTool("@UnCheckAllColumn_16x16.png", "Check/Uncheck All", new RoutedEventHandler(CheckUnCheckGridRow));
+                xDataSourceConfigGrid.AddToolbarTool("@UnCheckAllColumn_16x16.png", "Check/Unchecked All", new RoutedEventHandler(CheckUnCheckGridRow));
```

Comment:
revert it to Uncheck as its logical instead of unchecked

Reply:
done